### PR TITLE
Allow for multiple license files

### LINKED ISSUE #341: Allow for more than one license, including directories
When specifying multiple license files like below:
```yaml
  license: MIT
  license_family: MIT
  license_file:
    - LICENSE
    - NOTICE.md
```
`boa` throws the below error:
```
Recipe validation error

['LICENSE', 'NOTICE.md'] is not of type 'string'

Failed validating 'type' in schema['properties']['about']['properties']['license_file']:
    {'title': 'License File', 'type': 'string'}

On instance['about']['license_file']:
    ['LICENSE', 'NOTICE.md']
```

With `conda-build` multiple license files and also directories are allowed. This is important to be able to specify 3rd party licenses which should be included.

### PR DESCRIPTION
**Pull request by dhirschfeld**
Resolves #341

### CODE REVIEW

**Comment by wolfv** [comment]
Looks good to me! :)

**Comment by wolfv** [comment]
Looks like we need some fake licenses in that folder :)

**Comment by dhirschfeld** [comment]
```
ValueError: License file given in about/license_file (D:\a\boa\boa\tests\recipes\multiple_license\LICENSE) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

File: tests/recipes/multiple_license/LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (dhirschfeld) [inline tests/recipes/multiple_license/LICENSE:1]:
```suggestion
```

File: tests/recipes/multiple_license/NOTICE.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (dhirschfeld) [inline tests/recipes/multiple_license/NOTICE.md:1]:
```suggestion
```

**Pull request merged**